app/services/taxonomy/delete_page.rb

```diff
@@ -1,11 +1,14 @@
 module Taxonomy
   class DeletePage
-    delegate :content_id, to: :taxon
+    include ActiveModel::Model
```

Comment:
I don't quite follow this, this doesn't seem like a model like thing to me?

Reply:
It is used as a view model. The do_tag checkbox is rendered using simple_form, which takes a (view model and so it uses this one instead of Taxon (which it used previously). All relevant attributes are delegated to the Taxon model